index.js

```diff
@@ -132,19 +138,32 @@ importer = {
                 });
             }
 
+            // Create files map of path <> content if the data has content key
+            if (data[0].content) {
+                _.forEach(data, (file) => {
+                    var filePath = decodeURIComponent(path.resolve(file.fileName));
+                    filesMap[filePath] = file.content;
+                });
+            }
+
             async.each(rootSpecs, (rootSpec, cb) => {
-                var reader = new ramlParser.FileReader(function (path) {
+                var reader = new ramlParser.FileReader(function (filePath) {
                         return new Promise(function (resolve, reject) {
-                            var decodedFullPath = decodeURIComponent(path);
+                            var decodedFullPath = decodeURIComponent(filePath);
+
+                            // Only check this if the filesMap object is populated with the files content
+                            if (!_.isEmpty(filesMap)) {
```

Comment:
@umeshp7 Is it guaranteed that if one file has `content` prop then all of them will have `content` prop?

Reply:
Yes. That's the way I am sending the data from the app.